origami.json

```diff
@@ -5,11 +5,24 @@
     "support": "next.team@ft.com",
     "supportStatus": "experimental",
     "runServer": "true",
+    "demosDefaults": {
+        "sass": "demos/src/demo.scss",
+        "js": "demos/src/demo.js",
+        "documentClasses": "",
+        "dependencies": "o-fonts@^2.0.0,o-grid@^4.0.0"
+    },
     "demos": [
         {
-            "path": "/demos/light-signup-demo.html",
-            "expanded": true,
-            "description": ""
+            "name": "promo-image",
+            "description": "Light signup form with promo image",
+            "template": "demos/src/standard.html",
+            "path": "/demos/promo-image.html"
```

Comment:
I've just tested this and you are correct, it will automatically add the `path` to `origami.json`.
However would it be worth keeping it in the file? Otherwise every time the `make demo` command is run `origami.json` will be modified and updated.

This means the dev will have to manually ignore the changes when they are flagged up by git.